### ISSUE TITLE
Scrollspy issue 4.0.0

### ISSUE THREAD

**Issue by rasmuswoelk** [twbs/bootstrap#17677]
I'm having difficulties getting the scrollspy working in bootstrap 4.0.0.

Here is an example: http://pastebin.com/LJqUi4jX

I am using the bootstrap libraries from bootstraps own documentation directly. Also their own jquery version. In the bottom you can see the out-commented version of bootstrap3. If you activate it the scrollspy works.

I pretty sure I have done the (simple) installation correctly but I might be wrong. Anyone else having this issue?

**Comment by twbs-lmvtfy** [comment]
Hi @rasmuswoelk!

You appear to have posted a live example (http://pastebin.com/29f2mWuX), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 6, column 1 thru column 62: A ```` meta ```` element with an ```` http-equiv ```` attribute whose value is ```` X-UA-Compatible ```` must have a ```` content ```` attribute with the value ```` IE=edge ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Superseded by #17679.
